pkg/config/errors.go

```diff
@@ -33,6 +33,23 @@ func (e BpError) Unwrap() error {
 	return e.Err
 }
 
+// HintError is an error wrapper to suggest other variable names
+type HintError struct {
```

Comment:
Let's make `HintError` a general hint wrapper, not just for this particular type of error:
```go
typ HintError struct {
  Hint string
  Err error
}
```

Reply:
Should we change the string created as well?  (i.e. Should the Hint passed in be the new error message to be concatenated to the end of the current error?) (e.g. Hint = "Do you mean vars.test?"). Alternatively we could call it VarHintError to make it more specific and keep the "Do you mean %s" in the error.

Reply:
Addressed.